SCClassLibrary/Common/GUI/Base/QWindow.sc

```diff
@@ -172,9 +172,16 @@ Window {
 	onClose { ^view.onClose }
 	onClose_ { arg func; view.onClose_(func) }
 
-	*availableBottom {
-		^this.screenBounds.height - this.availableBounds.top - this.availableBounds.height
-	}
+	*lowestPosition {
+ 		^this.screenBounds.bottom - this.availableBounds.bottom
+ 	}
+
+	moveToBottom { |margin = 0|
+		this.bounds = this.bounds.moveTo(
+			this.bounds.left,
+			Window.lowestPosition + margin
+		)
+    	}
```

Comment:
indentation

Reply:
done in https://github.com/supercollider/supercollider/pull/6909/commits/92daf223de09e9a65112a913b24fc1ed8650d60c

Thanks!